build.sbt

```diff
@@ -13,3 +13,22 @@ scalacOptions in Test ++= Seq("-deprecation")
 testOptions in Test += Tests.Argument("-Djava.library.path=target/native")
 
 publishTo := Some(Resolver.file("file",  new File( "releases" )) )
+
+lazy val packageSqlite3C = taskKey[Unit]("Packages only Sqlite3C.java")
+
+packageSqlite3C := {
+  import scala.sys.process._
```

Comment:
Right now this task is stand-alone. Do you want it to get triggered by compile / package or trigger compile?

For latter, add `(compile in Compile).value` as the first line of the task.